docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -311,3 +516,70 @@ include::../utils/_class_hashes.adoc[]
 ==== `[.contract-item-name]#++constructor++#++(ref self: ContractState, public_key: felt252)++` [.item-kind]#constructor#
 
 Sets the account `public_key` and registers the interfaces the contract supports.
+
+[.contract]
+[[EthAccountUpgradeable]]
+=== `++EthAccountUpgradeable++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.8.0/src/presets/eth_account.cairo[{github-icon},role=heading-link]
```

Comment:
```suggestion
=== `++EthAccountUpgradeable++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.8.1/src/presets/eth_account.cairo[{github-icon},role=heading-link]
```
i think we need to update these, there's a bunch docs links and license comments with 0.8.0 while it should be 0.8.1

Reply:
Fixed!